modules/UI/videolayout/SmallVideo.js

```diff
@@ -374,17 +374,13 @@ SmallVideo.prototype.removeModeratorIndicator = function () {
 };
 
 /**
- * This is an especially interesting function. A naive reader might think that
- * it returns this SmallVideo's "video" element. But it is much more exciting.
- * It first finds this video's parent element using jquery, then uses a utility
- * from lib-jitsi-meet to extract the video element from it (with two more
- * jquery calls), and finally uses jquery again to encapsulate the video element
- * in an array. This last step allows (some might prefer "forces") users of
- * this function to access the video element via the 0th element of the returned
- * array (after checking its length of course!).
+ * Selects the video element.
+ * @return {jQuery|HTMLElement} a JQuery selector pointing to the video element.
  */
 SmallVideo.prototype.selectVideoElement = function () {
```

Comment:
I think it will be more accurate to name this function getVideoThumbnailElement or $videoThumbnailElement. Depending on the usage of this function we may also decide to get rid of the JQuery object and return a pure DOM element. The jsdoc should also be updated.